lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -458,7 +465,7 @@ def __init__(self, *args, zs=0, zdir='z', depthshade=True, **kwargs):
         super().__init__(*args, **kwargs)
         self.set_3d_properties(zs, zdir)
 
-    def set_sort_zpos(self, val):
+    def set_sort_zpos(self,val):
```

Comment:
no, there should be a space after the comma

Reply:
Absolutely, there was another `set_sort_zpos` where the space was missing, I added it then decided to revert PEP8 fixes, but looks like I reverted in the wrong function.